### PR TITLE
TOP DQM: Fixed MuonRelIso 

### PR DESCRIPTION
**Pull request by kuyoun**
#### PR description:

Fixed MuonRelIso (filled twice for miniAOD) 

Applied loose isloation cut for muon

#### PR validation:

Local test was done; cleaned unused lines and change minor things

### CODE REVIEW

**Comment by jfernan2** [comment]
@kuyoun I am assuming you are Youn Jung Roh since we don't have your github username identified in:
https://twiki.cern.ch/twiki/bin/viewauth/CMS/DQMContacts#TOP
Please include the PAG (TOP) name in the PR name and make a proper description of the PR: why do you need to change this and what changes you expect. 
Besides you have code style and quality issues to fix. Thanks

**Comment by kuyoun** [comment]
While do validation check, auto checker keeps pointed out "discrepancy on MuonRelIso for miniAOD".  It is because histo was filled twice.

**Comment by jfernan2** [comment]
please test

File: DQM/Physics/python/singleTopDQM_cfi.py [hunk 1]
```diff
@@ -181,11 +54,11 @@
       rho = cms.InputTag("fixedGridRhoFastjetAll"),
     ),                                     
     muonExtras = cms.PSet(                                               
-      select    = cms.string(looseMuonCut + " && pt>20 & abs(eta)<2.1"),                                              
+      select    = cms.string(tightMuonCut + " && pt>20 & abs(eta)<2.4"),
       isolation = cms.string(looseIsoCut)
     ),
     jetExtras = cms.PSet(
-      jetCorrector = cms.InputTag("dqmAk4PFCHSL1FastL2L3Corrector"),  #Use pak4PFCHSL1FastL2L3Residual for data!!!                                            
+      jetCorrector = cms.InputTag("ak4PFCHSL1FastL2L3Corrector"),  #Use pak4PFCHSL1FastL2L3Residual for data!!!
```
Comment (jfernan2) [inline DQM/Physics/python/singleTopDQM_cfi.py]: This is breaking again the issue found in https://github.com/cms-sw/cmssw/issues/22996
and fixed in https://github.com/cms-sw/cmssw/pull/28156
Please revert the change and test it getting the Tier0 configuration as explained in https://its.cern.ch/jira/browse/CMSTZ-427
Thanks

File: DQM/Physics/python/singleTopDQM_cfi.py [hunk 2]
```diff
@@ -262,11 +135,11 @@
       rho = cms.InputTag("fixedGridRhoFastjetAll"),
     ),                                     
     muonExtras = cms.PSet(                                               
-      select    = cms.string(looseMuonCut + " && pt>20 & abs(eta)<2.1"),                                              
+      select    = cms.string(tightMuonCut + " && pt>20 & abs(eta)<2.4"),                                              
       isolation = cms.string(looseIsoCut)
     ),
     jetExtras = cms.PSet(
-      jetCorrector = cms.InputTag("dqmAk4PFCHSL1FastL2L3Corrector"),  #Use pak4PFCHSL1FastL2L3Residual for data!!!                                            
+      jetCorrector = cms.InputTag("ak4PFCHSL1FastL2L3Corrector"),  #Use pak4PFCHSL1FastL2L3Residual for data!!!
```
Comment (jfernan2) [inline DQM/Physics/python/singleTopDQM_cfi.py]: This is breaking again the issue found in https://github.com/cms-sw/cmssw/issues/22996
and fixed in https://github.com/cms-sw/cmssw/pull/28156
Please revert the change and test it getting the Tier0 configuration as explained in https://its.cern.ch/jira/browse/CMSTZ-427
Thanks

File: DQM/Physics/python/topSingleLeptonDQM_cfi.py [hunk 3]
```diff
@@ -317,12 +55,13 @@
...
-    muonExtras = cms.PSet(                                               
-      select    = cms.string(looseMuonCut + " && pt>20 & abs(eta)<2.1"),                                              
+    muonExtras = cms.PSet(
+      select    = cms.string(tightMuonCut + " && pt>20 & abs(eta)<2.4"),                                               
+      #select    = cms.string(looseMuonCut + " && pt>20 & abs(eta)<2.4"),                                              
       isolation = cms.string(looseIsoCut)
     ),
     jetExtras = cms.PSet(
-      jetCorrector = cms.InputTag("dqmAk4PFCHSL1FastL2L3Corrector"),  #Use pak4PFCHSL1FastL2L3Residual for data!!!                                            
+      jetCorrector = cms.InputTag("ak4PFCHSL1FastL2L3Corrector"),  #Use pak4PFCHSL1FastL2L3Residual for data!!!
```
Comment (jfernan2) [inline DQM/Physics/python/topSingleLeptonDQM_cfi.py]: This is breaking again the issue found in https://github.com/cms-sw/cmssw/issues/22996
and fixed in https://github.com/cms-sw/cmssw/pull/28156
Please revert the change and test it getting the Tier0 configuration as explained in https://its.cern.ch/jira/browse/CMSTZ-427
Thanks

File: DQM/Physics/python/topSingleLeptonDQM_cfi.py [hunk 4]
```diff
@@ -512,11 +134,11 @@
...
     ),
     muonExtras = cms.PSet(
-      select     = cms.string(looseMuonCut + " & pt>20 & abs(eta)<2.1"),
-			isolation  = cms.string(looseIsoCut),
+      select     = cms.string(tightMuonCut + " & pt>20 & abs(eta)<2.4"),
+      isolation  = cms.string(looseIsoCut),
     ),
     jetExtras = cms.PSet(
-			jetCorrector = cms.InputTag("dqmAk4PFCHSL1FastL2L3Corrector"), #Use pak4PFCHSL1FastL2L3Residual for data!!!
+			jetCorrector = cms.InputTag("ak4PFCHSL1FastL2L3Corrector"), #Use pak4PFCHSL1FastL2L3Residual for data!!!
```
Comment (jfernan2) [inline DQM/Physics/python/topSingleLeptonDQM_cfi.py]: This is breaking again the issue found in https://github.com/cms-sw/cmssw/issues/22996
and fixed in https://github.com/cms-sw/cmssw/pull/28156
Please revert the change and test it getting the Tier0 configuration as explained in https://its.cern.ch/jira/browse/CMSTZ-427
Thanks

Comment (kuyoun) [reply DQM/Physics/python/topSingleLeptonDQM_cfi.py]: Local test with "topDQM_production_cfg.py" after changed to "dqmAk4PFCHSL1FastL2L3Corrector" ; but got error, not define "dqmAk4PFCHSL1FastL2L3Corrector". 
So, added line: process.load("DQMOffline.JetMET.jetMETDQMOfflineSource_cff")

----- Begin Fatal Exception 27-Mar-2020 06:43:11 CET-----------------------
An exception of category 'ProductNotFound' occurred while
   [0] Processing  Event run: 1 lumi: 31 event: 3005 stream: 0
   [1] Running path 'p'
   [2] Calling method for module ChainedJetCorrectorProducer/'dqmAk4PFCHSL1FastL2L3Corrector'
Exception Message:
Principal::getByToken: Found zero products matching all criteria
Looking for type: reco::JetCorrector
Looking for module label: ak4PFCHSL1FastjetCorrector
Looking for productInstanceName:

   Additional Info:
      [a] If you wish to continue processing events after a ProductNotFound exception,
add "SkipEvent = cms.untracked.vstring('ProductNotFound')" to the "options" PSet in the configuration.

Can you see why?

Comment (jfernan2) [reply DQM/Physics/python/topSingleLeptonDQM_cfi.py]: No sorry, I don't see why, I am afraid you should dig into the config
Thanks

**Comment by silviodonato** [comment]
@kuyoun are there any news?

**Comment by kuyoun** [comment]
couldn't find the solution yet. Do you know which script cotains "dqmAk4PFCHSL1FastL2L3Corrector"? I found one(DQMOffline.JetMET.jetMETDQMOfflineSource_cff), but it didn't work after load it.

**Comment by silviodonato** [comment]
@kuyoun is
https://cmssdt.cern.ch/lxr/search?%21v=CMSSW_11_1_X_2020-04-05-2300&_filestring=&_string=dqmAk4PFCHSL1FastL2L3Correcto
useful to you ?

**Comment by kuyoun** [comment]
Thank you for useful link. BTW, I checked we (TOP DQM) are only use "dqmAk4PFCHSL1FastL2L3Corrector", but I found many others used "ak4PFCHSL1FastL2L3Corrector".  What if we import ak4PFCHSL1FastL2L3Corrector from "DQM/Physics/python/topSingleLeptonDQM_cfi.py" and "DQM/Physics/python/singleTopDQM_cfi.py" such as adding 
from JetMETCorrections.Configuration.JetCorrectors_cff import ak4PFCHSL1FastL2L3Corrector

**Comment by jfernan2** [comment]
@kuyoun this is due to the fact that it was breaking as reported in the issue found in #22996
and fixed in PR #28156

**Comment by jfernan2** [comment]
please test

**Comment by jfernan2** [comment]
+1

**Comment by silviodonato** [comment]
+1
